schedule/yast/autoyast_create_hdd_gnome_RAID_encrypted_home.yaml

```diff
@@ -0,0 +1,22 @@
+name: autoyast_create_hdd_gnome_RAID_encrypted_home
```

Comment:
this mistake was introduced in many places in the past, please use lower letter to name files and this kind of things, 'raid'. When used in description value is fine, but for naming assets or programming function doesn't make sense, it lead us to silly mistakes not finding things.

Reply:
Replaced RAID as lowercase from all name files.